set.mm

```diff
@@ -11796,21 +11796,40 @@ This definition (in the form of ~ dfifp2 ) appears in Section II.24 of
     ( wnor wo wn notnotb df-nor notbii nornot 3bitr2ri bicomi ) ABCZLCZABDZNNEZ
     ELEMNFLOABGHLIJK $.
 
+  $( This lemma of ~ norass connects the bi-conditional to implications in a
+     different way than ~ dfbi2 does.  (Contributed by Wolf Lammen,
+     18-Dec-2023.) $)
+  norass-lem1 $p |- ( ( ph <-> ps ) <-> ( ( ph -> ps ) <-> ( ps -> ph ) ) ) $=
+    ( wb wi wa dfbi2 pm5.1 sylbi impbi wn conax1 pm5.21im syl2imc bija impbii )
+    ABCZABDZBADZCZPQRESABFQRGHQRPABIRJAJQJBJPBAKABKABLMNO $.
```

Comment:
To me, this statement is interesting in its own right, so you could move it much earlier, with a specific label. `dfbi4` ?

Reply:
Unfortunately it is not a definition, since <-> appears on both sides.  The name should not start with df.

Reply:
I decided on impimprbi. To avoid any confusion with biimp/biimpr the bi was suffixed.